phase1/gce/gce.jsonnet

```diff
@@ -161,15 +161,18 @@ function(cfg)
             "k8s-kubeadm-kubernetes-version": "%(kubernetes_version)s" % p2,
             "k8s-kubeadm-kubelet-version": "%(kubelet_version)s" % p2,
             "k8s-kubeadm-enable-cloud-provider": (if std.objectHas(p2, "enable_cloud_provider") && p2.enable_cloud_provider then "true" else "false"),
+            "k8s-kubeadm-feature-gates": if std.objectHas(p2.kubeadm, "feature_gates") then "%(feature_gates)s" % p2.kubeadm else "",
           } else {
             "k8s-config": config_metadata_template % [names.master_ip, "master"],
             "k8s-ca-public-key": "${tls_self_signed_cert.%s-root.cert_pem}" % p1.cluster_name,
             "k8s-apisever-public-key": "${tls_locally_signed_cert.%s-master.cert_pem}" % p1.cluster_name,
             "k8s-apisever-private-key": "${tls_private_key.%s-master.private_key_pem}" % p1.cluster_name,
             "k8s-master-kubeconfig": kubeconfig(p1.cluster_name + "-master", "local", "service-account-context"),
           },
-          disk: [{
-            image: gce.os_image,
+          boot_disk: [{
```

Comment:
yes, I noticed it. So I modified it too .. but I could not deploy with that second occurence as boot-disk. I guess that has to do that one is used directly for an "instance" and the other one is a "template".

Looking into the doc of gcloud create instance, 'disk' (as it was before) should be accepted, and there is no 'boot_disk' option. However on my test (with GCE) does work only adding the option boot_disk.

I will re-check that part.

Reply:
see here in terraform.io :

create instance : https://www.terraform.io/docs/providers/google/r/compute_instance.html
create instance template : https://www.terraform.io/docs/providers/google/r/compute_instance_template.html

first one work with "boot_disk" and second one with "disk".

Reply:
NOTE: I am not sure since when that option changed. "boot_disk" is now required. And I can validate that w/o that option the instance could not be created ...

but : how this configuration for GCE was working so far ? .... maybe recent change in the version of terraform.